index.src.html

```diff
@@ -806,22 +757,59 @@
 The authenticator MUST return both the |authenticatorData| and the raw signature back to the client.
 
 
-## Key Attestation Format ## {#attestation}
+## Credential Attestation Statements ## {#attestation}
 
 An attestation statement is a specific type of signature, which contains statements about a credential itself and the
 authenticator that holds it. Therefore, the format of attestation statements and the procedures for generating them closely
 parallel those for generating WebAuthn assertions as described in [[#signature-format]], though the semantics of the contextual
 bindings are quite different.
 
-The general structure of an attestation statement is specified in [[#iface-attestation-statement]]. This section provides a
-profile of these structures when different types of hardware act as the crypto kernel. More profiles are expected to be added as
-the specification evolves.
+This specification defines a number of types of attestation statements. All of these types share the following format:
+
+<pre class="idl">
+    interface AttestationStatement {
+        readonly    attribute unsigned long version;
+        readonly    attribute DOMString     claimedAAGUID;
+        readonly    attribute DOMString[]   x5c;
+        readonly    attribute DOMString     alg;
+        readonly    attribute DOMString     rawData;
+        readonly    attribute DOMString     signature;
+    };
+</pre>
+
+<div dfn-for="AttestationStatement">
+    The <dfn>version</dfn> member specifies the version number of the rawData object.
+
+    The <dfn>claimedAAGUID</dfn> element contains the claimed Authenticator Attestation GUID (a version 4 GUID, see
+    [[RFC4122]]). This value is used by the <a>[RP]</a> to look up the trust anchor for verifying the following signature. If
+    the verification succeeds, the AAGUID related to the trust anchor is trusted. This field MUST be present, if either no
+    attestation certificates are used (e.g., DAA) or if the attestation certificate doesn't contain the AAGUID value in an
+    extension.
+
+    The <dfn>x5c</dfn> attribute contains the attestation certificate and its certificate chain as described in [[!RFC7515]]
+    section 4.1.6.
+
+    The <dfn>alg</dfn> element contains the name of the algorithm used to generate the attestation signature according to
+    [[!RFC7518]]. See [[#packed-attestation-signature]] for the signature algorithms to be implemented by servers.
```

Comment:
I believe that is correct (I did not add that line, just moved it). However I can certainly add that detail. 

Also, I also notice that the reference in the next link (also merely moved) seems out of place. Maybe that sentence should just be removed, and we can let the individual attestation types specify what must be implemented by servers. WDYT?